docker-compose/init-compose.sh

```diff
@@ -19,7 +19,9 @@ for run in {1..60}; do
   sleep 0.1 
   temporal  operator search-attribute  create --name IwfExecutingStateIds --type KeywordList
   sleep 0.1
-  if checkExists "IwfWorkflowType" ] && checkExists "IwfGlobalWorkflowVersion" && checkExists "IwfExecutingStateIds" ] ; then
+  temporal  operator search-attribute  create --name CustomKeywordArrayField --type KeywordList
```

Comment:
You don't need to add it here, other than that this looks good 👍

Reply:
init-compose.sh is used if we want to setup the docker containers, but run iwf-server locally to test local changes. If some one runs the server locally then runs java-sdk tests against it the tests will fail.

Reply:
Ah I see, makes sense! Thanks for clarifying, appreciate it! 🙌